Rakefile

```diff
@@ -32,3 +31,12 @@ task :run => :compile do
   system(RbConfig.ruby, *%w(-I ./lib test.rb))
 end
 
+task :test_dap do
+  ENV['RUBY_DEBUG_DAP_TEST'] = '1'
+end
+
+Rake::TestTask.new(:test_dap) do |t|
+  t.libs << "test"
```

Comment:
Based on what I tested in https://github.com/ruby/debug/pull/462, these `libs` assignment aren't necessary.